shell/platform/windows/platform_handler.cc

```diff
@@ -205,6 +203,18 @@ int ScopedClipboard::SetString(const std::wstring string) {
 
 }  // namespace
 
+static const char* exit_type_names[] = {"required", "cancelable"};
```

Comment:
Please add a comment that this must be kept in sync with `platform_handler.h`'s `ExitType`

Reply:
Alternatively, turn this into a switch on the enum and inline that into `RequestAppExit`